packages/app/src/apis.ts

```diff
@@ -29,6 +30,11 @@ export const apis: AnyApiFactory[] = [
     deps: { configApi: configApiRef },
     factory: ({ configApi }) => ScmIntegrationsApi.fromConfig(configApi),
   }),
+  createApiFactory({
```

Comment:
Here again, is this really needed or couldn't we read the data with the config api?